benchmark/v4.0-performance-benchmarking-with-tpch.md

```diff
@@ -0,0 +1,155 @@
+---
+title: TiDB TPC-H 性能对比测试报告 - v4.0 对比 v3.0
+category: benchmark
+draft: true
```

Comment:
```suggestion
draft: false
```

should it be false?

Reply:
Now tpcc benchresult is true, I just keep that for consistent

Reply:
@shuke987 @lilin90 please help us to confirm whether it should be `false`.

Reply:
If the testing results in this doc are ready to publish, then we can remove this `draft` line. Just keep it when the content is not ready.

Reply:
Removed.